views/shop/renames.tt

```diff
@@ -0,0 +1,88 @@
+[% sections.title =  dw.ml( ".title" ) %]
+[%- dw.need_res( { group => "foundation" },
+    ## date
+    "js/vendor/pickadate.js/picker.js"
+    "js/vendor/pickadate.js/picker.date.js"
+    "js/vendor/pickadate.js/picker.time.js"
+    "js/vendor/pickadate.js/legacy.js"
+    "stc/css/components/pickadate/datetime.css"
+    "js/jquery.shop.js"
+
+) -%]
+
+<p><a href='[% site.root %]/shop'>&lt;&lt; [% dw.ml( '.backlink', { sitename => site.nameshort } )  %]</a></p>
+
+[% cart_display %]
+
+<p>[% dw.ml( ".intro.$for" ) %]</p>
+<p>[% dw.ml( '.action', { aopts => "href='/rename'" } )  %]</p>
+
+<div style='clear: both;'></div>
+<form method='post'>
+[% dw.form_auth() %]
+
+[% IF for == "gift" %]
+<div class="row">
+<div class="columns large-6 end">
+<div class="row collapse">
+    <div class="columns medium-6">
+        <label for="username">[% dw.ml( '.giftfor.username') %]</label>
+    </div>
+    <div class="columns medium-6">
+        [% form.textbox(  name => 'username')  %]
+    </div>
+</div>
+<div class="row collapse" id="deliverydate">
+    <div class="columns medium-6">
+        <label for="deliverydate" class="inline">[% dw.ml( '.giftfor.deliverydate') %]</label>
+    </div>
+    <div class="columns medium-5 small-10">
+        [%- form.textbox(
+            name = "deliverydate"
+            id = "js-deliverydate"
+
+            maxlength = "10"
+            size = "10"
+
+
+            default = date.iso8601.substr(0, 10)
```

Comment:
I think the templates for account and renames might also benefit from shared includes to avoid duplication.